catalogs/catalog-jdbc-common/src/test/java/com/datastrato/gravitino/catalog/jdbc/operation/SqliteDatabaseOperations.java

```diff
@@ -91,4 +91,9 @@ public String generateCreateDatabaseSql(
   public String generateDropDatabaseSql(String databaseName, boolean cascade) {
     return null;
   }
+
+  @Override
+  protected void validateDatabaseName(String databaseName) {
+    // do nothing
+  }
```

Comment:
We may need to do something here.

Reply:
Oh sorry, I thought sqlite is not in use for now. 
I am not familiar with sqlite. From what I have researched, sqlite does not impose any naming requirements or length limitations. Lots of special characters and white space are supported. Do you have any suggestions on what we should check? 
Thank you!

Reply:
SQLite database names are filenames, so the same restrictions for a filename would apply.

Reply:
That seems like a good approach to me, go for it.

Reply:
I added checks for SQLite database name in https://github.com/datastrato/gravitino/pull/2335/commits/17a14b1605fd3b24e5e12775f9f5356e523166f4 and refined PR per comments. Please take a look when you have time.

Reply:
I thnk we also need to validate the database name on create not just delete

Reply:
And also, check in the other delete method.

Reply:
Sorry for this late reply. I've been busy with work these days.
Since database naming convention is similar to that of a table, I think we can create four utils classes `xxxValidateNamingUtils` in the following paths?
- com.datastrato.gravitino.catalog.jdbc.utils.ValidateNamingUtils
- com.datastrato.gravitino.catalog.jdbc.utils.SqliteValidateNamingUtils
- com.datastrato.gravitino.catalog.jdbc.utils.MysqlValidateNamingUtils
- com.datastrato.gravitino.catalog.jdbc.utils.PostgreSqlValidateNamingUtils

In `ValidateNamingUtils`, we might have a `validateName` function, which can be shared within table and database create/delete operations.
The `validateDatabaseName` in mySQL, Postgres, and SQLite TableOperation or DatabaseOperation can then call relevant utils class. The `validateDatabaseName` will then be called by delete or create function.
With this, I think we may reduce duplicated logic. 
What do you think about this approach? 
We may discuss the naming of util class as well. Please let me know your thought, thank you!

Reply:
Hi @justinmclean @yuqi1129, should I apply the validation rule to table create and delete as well per the previous comment? Or we should merge this and file another PR to deal with table naming?

Reply:
If you can work the tables, go ahead in this PR, but the longer the PR is open, the more likely it will get conflicts.

Reply:
I see. With that in mind, I think I will open another PR for table operations. Rebased main for this one.